modules/ROOT/pages/promote-api-task.adoc

```diff
@@ -7,13 +7,13 @@ A tight integration of Anypoint Platform components extends the use of environme
 
 *Note:* You must have Manage APIs Configuration permission to promote an API. In addition you must have Access Management permission to set permissions for viewing and managing an entity. You can only promote an entity (Policies, Alerts, etc.) if you have permission to view the source and permission to manage the destination over each entity. Typical view permissions are View API Configuration and View Policies as explained in xref:environment-permission-task.adoc[To Grant Permission to Access Environments]. Set the permissions as needed before proceeding.
```

Comment:
This text needs editing, but I am leaving it because it is not changed. However, I wonder if the formatting of the heading Note is correct?

Reply:
Agree that this topic needs editing. I'll tackle this in another PR.